src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -514,7 +514,7 @@ def compute_sliding_rp_violations(
 )
 
 
-def get_synchrony_counts(spikes, synchrony_sizes, all_unit_ids):
+def get_synchrony_count(spikes, synchrony_sizes, all_unit_ids):
```

Comment:
can you revert this? I think the naming is correct because it returns multiple counts for different synchrony sizes. (plus all tests are failing!!!)
```suggestion
def get_synchrony_counts(spikes, synchrony_sizes, all_unit_ids):
```